### PR TITLE
Schema features & fixes: annotation object id UniProtKB support, annotation publications, microscope manufacturer enum

### PR DESCRIPTION
**Pull request by daniel-ji**
- Fix annotation publications list description (include PDB) and also have "PDB-" prefix, not "pdb" prefix
- Add "SIMULATED" option to microscope manufacturer enum
- Add UniProtKB id support to dataset.annotation_object.id
- Change `default_range` of v2.0.0 common.yaml schema to Any, so that `range` / type values of slots are properly written.

### CODE REVIEW

File: schema/ingestion_config/v1.0.0/ingestion_config_models_extended.py [hunk 1]
```diff
@@ -321,7 +346,8 @@ def validate_id_name_object(
 
     logger.debug("Valid ID, now checking if name '%s' matches ID: %s", name, id)
 
-    valid_name = any(name == retrieved_name for retrieved_name in retrieved_names)
+    # if the retrieved names is empty, we can assume the name is valid
```
Comment (Bento007) [inline schema/ingestion_config/v1.0.0/ingestion_config_models_extended.py:349]:
```suggestion
    # if the retrieved names are empty, we can assume the name is valid
```

File: schema/ingestion_config/v1.0.0/ingestion_config_models_extended.py [hunk 2]
```diff
@@ -406,6 +432,8 @@ class ExtendedValidationDepositionKeyPhotoSource(DepositionKeyPhotoSource):
 # ==============================================================================
 @alru_cache
 async def lookup_doi(doi: str) -> Tuple[str, bool]:
+    # Remove the doi: prefix if it exists
```
Comment (Bento007) [inline schema/ingestion_config/v1.0.0/ingestion_config_models_extended.py:435]: nit: I don't think this comment adds any additional clarity to the code. I suggest removing.

File: schema/ingestion_config/v1.0.0/ingestion_config_models_extended.py [hunk 3]
```diff
@@ -427,6 +455,8 @@ async def lookup_emdb(emdb_id: str) -> Tuple[str, bool]:
 
 @alru_cache
 async def lookup_pdb(pdb_id: str) -> Tuple[str, bool]:
+    # Strip the PDB- prefix
```
Comment (Bento007) [inline schema/ingestion_config/v1.0.0/ingestion_config_models_extended.py:458]: nit: I don't think this comment adds any additional clarity to the code](nit: I don't think this comment adds any additional clarity to the code. I suggest removing.)

File: schema/ingestion_config/v1.0.0/ingestion_config_models_extended.py [hunk 4]
```diff
@@ -436,20 +466,18 @@ async def lookup_pdb(pdb_id: str) -> Tuple[str, bool]:
...
 async def validate_publication_lists(publication_list: List[str]) -> List[str]:
     tasks = []
 
     for publication in publication_list:
-        for publication_type, (regex, validate_function) in PUBLICATION_REGEXES_AND_FUNCTIONS.items():
+        for _, (regex, validate_function) in PUBLICATION_REGEXES_AND_FUNCTIONS.items():
             if not re.match(regex, publication):
                 continue
-            # edge case for DOI, remove the doi: prefix
-            updated_publication = publication.replace("doi:", "") if publication_type == "doi" else publication
```
Comment (manasaV3) [inline schema/ingestion_config/v1.0.0/ingestion_config_models_extended.py:451]: Curious on why this was removed?

**Review by manasaV3 — APPROVED** [review]
LGTM

**Pull request merged**